src/server/routes/api/document/index.js

```diff
@@ -319,6 +320,44 @@ let getSbgnFromTemplates = templates => {
     .then(handleResponseError);
 };
 
+/**
+ * generateSitemap
+ * Create sitemap xml data for documents with PUBLIC status
+ *
+ * @return an sitemap conforming to [sitemaps.org]{@link https://www.sitemaps.org/protocol.html}
+ */
+const generateSitemap = () => {
+  let status = DOCUMENT_STATUS_FIELDS.PUBLIC;
+  let tables;
+
+  return (
+    tryPromise( () => loadTables() )
+    .then( tbls => {
+      tables = tbls;
+      return tables;
+    })
+    .then( tables => {
+      let t = tables.docDb;
+      let { table, conn, rethink: r } = t;
+      let q = table;
+
+      q = q.filter( r.row( 'status' ).eq( status ) );
+      q = q.orderBy( r.desc( 'createdDate' ) );
+      q = q.pluck( ['id', 'secret'] );
+      return q.run( conn );
+    })
+    .then( cursor => cursor.toArray() )
+    .then( res => {
+      return Promise.all( res.map( docDbJson => {
+        let { id } = docDbJson;
+        let docOpts = _.assign( {}, tables, { id } );
+        return loadDoc( docOpts ).then( getDocJson );
+      }));
+    } )
+    .then( docs2Sitemap )
```

Comment:
Maybe append the home page here?

Reply:
It's already appended in the sitemap.js code. Isn't that OK?